content/chainguard/libraries/python/overview.md

```diff
@@ -76,31 +98,100 @@ In some cases, multiple CVEs may be remediated in a specific library version.
 For example, `aiohttp` has fixes for both  CVE-2024-23334 and CVE-2024-30251 in
 the version `3.9.1+cgr.2`.
 
-## Technical Details
+<a id="requirements"></a>
 
-Most organizations consume Chainguard Libraries for Python through a repository
-manager such as Cloudsmith, JFrog Artifactory or Sonatype Nexus Repository. For
-full details, refer to our [Global Configuration
-documentation](/chainguard/libraries/python/global-configuration/). The rest of
-this article provides details of the underlying implementation of Chainguard
-Libraries for Python and how to access individual libraries manually.
+## Runtime and Build Requirements
 
-The Chainguard Libraries for Python indexes use the PyPI repository format and 
-only include release artifacts of the libraries built by Chainguard from source.
+The runtime requirements for Python artifacts available from Chainguard
+Libraries for Python are identical to the requirements of the original upstream
+project. For example, if a Python wheel retrieved from PyPI requires Python 3.10
+or higher, the same Python 3.10 runtime requirement applies to the binary
+artifact from Chainguard Libraries for Python.
 
-The URLs for the repositories are:
+All packages are distributed as Python wheels, following the [Python binary
```

Comment:
```suggestion
Chainguard distributes all packages as Python wheels, following the [Python binary
```

trying to cut out passive voice where I can, not sure this is the right move here though